scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -107,6 +108,70 @@ def result(self) -> List[str]:
 
 # BEGIN MERGE SLOT 6
 
+class BignumModRawConvertRep(bignum_common.ModOperationCommon,
+                             BignumModRawTarget):
+    # This is an abstract class, it's ok to have unimplemented methods.
+    #pylint: disable=abstract-method
+    """Test cases for representation conversion."""
+    arity = 1
+
+    def __init__(self, val_n: str, val_a: str, bits_in_limb: Optional[int],
+                 rep: bignum_common.ModulusRepresentation) -> None:
+        if bits_in_limb is None:
+            super().__init__(val_n=val_n, val_a=val_a)
+        else:
+            self.input_style = "arch_split"
+            super().__init__(val_n=val_n, val_a=val_a, bits_in_limb=bits_in_limb)
+        self.rep = rep
+
+    def arguments(self) -> List[str]:
+        return ([bignum_common.quote_str(self.arg_n), self.rep.symbol(),
+                 bignum_common.quote_str(self.arg_a)] +
+                self.result())
+
+    def description(self) -> str:
+        base = super().description()
```

Comment:
I agree that it's iffy, but I didn't want to repeat the description format so that the parent class could change the format if it wants. In the long term, I'd like to have a way to inject the information.